doc/Learn/Markup/Appendix/AccessingControlInstance.md

```diff
@@ -21,3 +21,50 @@ button.Click += delegate {
     button.Content = $"Clicked {i++} times";
 }
 ```
+
+You can also define a CallBack using the `Name` method:
```

Comment:
Is CallBack a C# Markup term for something? 

We should simplify it to just say you can provide an `Action` or a `delegate` method or something.

cc: @dansiegel for final decision

Reply:
> Is CallBack a C# Markup term for something?

No... and I would agree... @eriklimakc depending on the context I would use `delegate` or `Action` to describe what we're doing. Technically it's not a callback since we're not responding to an event, and we're really invoking a delegate action.

Reply:
Thanks. I just pushed a commit with the changes.